terraform-runner/README.MD

```diff
@@ -0,0 +1,24 @@
+# Introduction
+
+This docker container contains the [UniPipe CLI](https://github.com/meshcloud/unipipe-service-broker/tree/master/cli) 
+and schedules `unipipe terraform` every minute via a cron job. It can be used in combination with the 
+[UniPipe Service Broker](https://github.com/meshcloud/unipipe-service-broker) to actually process the service instances 
+that were written by the UniPipe Service Broker.
+
+# How to run
+
+At first you have to create an `env.list` and set some mandatory environment variables for the docker container.
+
+```
+GIT_USER_EMAIL=unipipe-terraform@meshcloud.io
+GIT_USER_NAME=UniPipe Terraform Runner
+GIT_REPO=<git repository ssh url>
```

Comment:
t: unipipe-service-broker calls this `GIT-REMOTE` (see https://github.com/meshcloud/unipipe-service-broker/wiki/Reference#configuration-reference)

Reply:
unipipe-service-broker also separates the remote url and the branch, but for this docker container here, i'd go with only the complete url. There is no need to separate it. But also in that case it is valid to call it GIT_REMOTE. Changed it accordingly